### DISCUSSION TITLE
Extract but its vey slow

### DISCUSSION THREAD

**Discussion by bilimirror** [anasty17/mirror-leech-telegram-bot#1257] [Q&A] [answered]
i use command /unzipleech, and its finish download, and then process extract
But, its very slow, cpu usage 0,1%

My vps
Centos7 1 core 1gb 
![Screenshot 2022-11-30 151339](https://user-images.githubusercontent.com/106572101/204742669-b5ead8b7-fcec-4e82-9dd4-5c193140d001.png)

on my other vps with ubuntu, its totally fine so issue

**Comment by anasty17** [comment]
Ok what i should do ?

**Reply by bilimirror** [reply]
any suggestions to solve this problem?

**Comment by bilimirror** [comment]
Here log.txt
[log.txt](https://github.com/anasty17/mirror-leech-telegram-bot/files/10122283/log.txt)

**Comment by bilimirror** [comment, marked as answer]
it fix, i forget config something

**Comment by bilimirror** [comment]
your bot its awesome, thanks😁😁😁
